core/src/main/java/org/mskcc/cbio/portal/dao/GeneticAlterationUtil.java

```diff
@@ -47,9 +51,19 @@
  *
  * @author Ethan Cerami.
  */
+@Component
 public class GeneticAlterationUtil {
     private static final String NAN = "NaN";
 
+    private static MutationRepository mutationRepository;
+    private static MutationModelConverter mutationModelConverter;
+
+    @Autowired
```

Comment:
Spring will autowire the static fields during the application startup, so the relying classes don't have to be managed by Spring.